tsdb/head.go

```diff
@@ -1609,6 +1619,22 @@ func (h *Head) getOrCreateWithID(id chunks.HeadSeriesRef, hash uint64, lset labe
 	return s, true, nil
 }
 
+func (h *Head) mmapHeadChunks() {
```

Comment:
This function can use a comment explaining what it is doing.

Reply:
Added